PostmanTests/UserService.postman_collection.json

```diff
@@ -50,6 +50,144 @@
 				}
 			],
 			"protocolProfileBehavior": {}
+		},
+		{
+			"name": "CreateUser",
+			"item": [
+				{
+					"name": "With correct body",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "6e99293a-94f5-4726-9781-a4a80260899b",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"pm.test(\"Body matches string\", function () {",
+									"    pm.expect(pm.response.text()).to.include(\"true\");",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"email2@yandex.ru\",\n\t\"Password\" : \"123\",\n\t\"FirstName\" : \"Lep\",\n\t\"LastName\" : \"Kor\",\n\t\"Birthday\" : \"2020-04-05\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://localhost:5011/users/create",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"port": "5011",
+							"path": [
+								"users",
+								"create"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Body without password",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "39e85dc7-4cf3-474f-b21f-967a65d24e72",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(500);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"email@yandex.ru\",\n\t\"FirstName\" : \"Leo\",\n\t\"LastName\" : \"Kor\",\n\t\"Birthday\" : \"2020-04-05\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://localhost:5011/users/create",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"port": "5011",
+							"path": [
+								"users",
+								"create"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Body with empty fields",
```

Comment:
If only one field will be missed who have to check this? Unit or postman tests?

Reply:
An extra test won't hurt. This test is to verify that the validator is working.However, it is not merged yet